src/app/client/src/app/modules/workspace/components/collaboration-content-filter/collaboration-content-filter.component.ts

```diff
@@ -88,7 +100,7 @@ export class CollaborationContentFilterComponent implements OnInit {
  */
   constructor(resourceService: ResourceService, config: ConfigService,
     activatedRoute: ActivatedRoute,
-    route: Router) {
+    route: Router,telemetryService: TelemetryService) {
     this.route = route;
```

Comment:
this. telemetryService = telemetryService;

Reply:
Added